### PR TITLE
Third feature added

### PR DESCRIPTION
**Pull request by hnavar**

### CODE REVIEW

**Comment by hnavar** [comment]
Yey it werked

File: server/api/movies.ts [hunk 1]
```diff
@@ -85,6 +87,68 @@ MoviesRouter.post('/', (req: Request, res: Response) => {
...
+
+   MoviesRouter.get('/moviesByRatingNC-17', (req: Request, res: Response) => {
+  grabMovieIdWithRating("NC-17")
+     .then((data: any) => {
+       res.send(data);
+     }).catch((error: any) => {
+       console.log(error);
+       res.status(500).end();
+     });
+   });
```
Comment (badams92) [inline server/api/movies.ts:128]: All of these `movieByRating` GET handlers should be combined into one function.  Use a parameter.

File: server/api/movies.ts [hunk 2]
```diff
@@ -85,6 +87,68 @@ MoviesRouter.post('/', (req: Request, res: Response) => {
...
+    //  console.log(req.body.name);
+    return grabActorOrDirectorID(req.body.name)
+        .then((data: any) => {
+          return data;
+        })
+        .then((data: any) => {
+       return grabMoviesByActorsOrDirectors(data);
+     }).then((data: any) => {
+      //  console.log(data);
+       res.send(data);
```
Comment (badams92) [inline server/api/movies.ts:140]: Don't forget the HTTP status code

**Comment by hnavar** [comment]
I updated my components to make them more dynamic and fix bugs.

**Comment by carlbenjaminlyon** [comment]
Good deal. Just remember to get the refactor on the age rating components.

**Pull request merged**